packages/components-js/tests/e2e/puppeteer/specs/checkbox-wrapper.e2e.ts

```diff
@@ -136,6 +140,78 @@ it('should toggle checkbox when input is clicked', async () => {
   expect(checkedImage).not.toBe(await getBackgroundImage(input));
 });
 
+it('should not toggle checkbox when checkbox wrapper is clicked in loading state', async () => {
+  await setContentWithDesignSystem(
```

Comment:
You could have extended the `initCheckbox` helper by the loading option instead of duplicating the test setup several times.